gslib/utils/copy_helper.py

```diff
@@ -2968,7 +2969,7 @@ def _DownloadObjectToFile(src_url,
             'typically happens when using gsutil to download from a subdirectory '
             'created by the Cloud Console (https://cloud.google.com/console)' %
             dst_url.object_name)))
-    return (0, 0, dst_url, '')
+    raise InvalidUrlError('Invalid destination path: %s' % dst_url.object_name)
```

Comment:
What's the point of the warning if we're going to raise an error?
Maybe the warning text should be merged into the error?

Reply:
Error may get ignored if you are using parallel processing or if you are using the continue-on-error flag.

Reply:
Ahh, maybe add a code comment about that

Reply:
Done.